field_map.go

```diff
@@ -39,6 +40,7 @@ func (t tagSort) Less(i, j int) bool { return t.compare(t.tags[i], t.tags[j]) }
 type FieldMap struct {
 	tagLookup map[Tag]field
 	tagSort
+	rwLock *sync.RWMutex
```

Comment:
`minor`: It's usually named `mu`, so I suggest using that

Reply:
nvm, missed your comment in PR description that we are simply copying the commit from original repo.